src/guides/v2.3/howdoi/custom-attributes/text-field.md

```diff
@@ -0,0 +1,346 @@
+---
+layout: tutorial
+group: how-do-i
+subgroup:
+title: Text Field Custom Attribute
+menu_node:
+level3_subgroup: custom-attributes
+menu_order: 2
+contributor_name: Adarsh Manickam
+contributor_link: https://github.com/drpayyne
+---
+
+## Overview
+
+This tutorial describes how a developer can create a custom text field attribute for the Customer entity using code. This will reflect in both the Magento Admin's [Customer Grid](https://docs.magento.com/user-guide/customers/customer-account-manage.html) and [Customer Form](https://docs.magento.com/user-guide/customers/customer-account-update.html).
+
+This Customer attribute will be used to save and view the customer's ID in an external system, as an example. It will be created as an EAV attribute in a data patch. The EAV model allows a developer to add custom functionality to the Magento entities without modifying the core databases and schemas. Data patches are run just once, so this code will run just once for the custom attribute to be created and never run again causing issues.
```

Comment:
```suggestion
This Customer attribute will be used to save and view the customer's ID in an external system, as an example. It will be created as an EAV attribute in a data patch. The EAV model allows a developer to add custom functionality to the Magento entities without modifying the core databases and schemas. Data patches are run just once, so this code will create the custom attribute and will never run again, which could cause issues.
```